ci/cloudbuild/builds/check-api.sh

```diff
@@ -51,16 +59,11 @@ function dump_abi() {
 }
 export -f dump_abi # enables this function to be called from a subshell
 
-libraries=(
-  "google_cloud_cpp_bigquery"
-  "google_cloud_cpp_bigtable"
-  "google_cloud_cpp_common"
-  "google_cloud_cpp_grpc_utils"
-  "google_cloud_cpp_iam"
-  "google_cloud_cpp_spanner"
-  "google_cloud_cpp_storage"
-  "google_cloud_cpp_pubsub"
+feature_list+=(
+  "common"
+  "grpc_utils"
 )
+mapfile -t libraries < <(for f in "${feature_list[@]}"; do echo "google_cloud_cpp_${f}"; done)
```

Comment:
No change necessary, but it looks like this should work similar to what you did above using `printf`, like `... <(printf "google_cloud_cpp_%s\n" "${feature_list[@]}")`